config/environments/development.rb

```diff
@@ -0,0 +1,76 @@
+require "active_support/core_ext/integer/time"
```

Comment:
Whats this file about?

Reply:
How do we know when it is running in `development` environment?

Reply:
Rails gets its environment from the `RAILS_ENV` shell variable when the server starts. We can use the command `RAILS_ENV=development rails s` to explicitly define to start the server with a `development` environment.